### PR TITLE
zh-trans: modify content/zh/docs/concepts/workloads/controllers/statefulset.md

### PR DESCRIPTION
**Pull request by lpf7551321**
zh-trans: modify content/zh/docs/concepts/workloads/controllers/statefulset.md
Update: content/zh/docs/concepts/workloads/controllers/statefulset.md
Co-Authored-By: lpf7551321

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/statefulset.md [hunk 1]
```diff
@@ -250,14 +258,15 @@ This must be done manually.
...
-Kubernetes 为每个 VolumeClaimTemplate 创建一个 [PersistentVolume](/docs/concepts/storage/persistent-volumes/)。在上面的 nginx 示例中，每个 Pod 将会得到基于 StorageClass `my-storage-class` 提供的 1 Gib 的 PersistentVolume。如果没有声明 StorageClass，就会使用默认的 StorageClass。当一个 Pod 被安排到节点上时，它的 `volumeMounts` 挂载了与其 PersistentVolumeClaims 相关联的 PersistentVolume。请注意，当 Pod 或者 StatefulSet 被删除时，与PersistentVolumeClaims 相关联的 PersistentVolume 并不会被删除。要删除它必须通过手动方式来完成。
+Kubernetes 为每个 VolumeClaimTemplate 创建一个 [PersistentVolume](/docs/concepts/storage/persistent-volumes/)。在上面的 nginx 示例中，每个 Pod 将会得到基于 StorageClass `my-storage-class` 提供的 1 Gib 的 PersistentVolume。如果没有声明 StorageClass，就会使用默认的 StorageClass。当一个 Pod 被安排到节点上时，它的 `volumeMounts` 挂载了与其 PersistentVolumeClaims 相关联的 PersistentVolume。请注意，当 Pod 或者 StatefulSet 被删除时，与 PersistentVolumeClaims 相关联的 PersistentVolume 并不会被删除。要删除它必须通过手动方式来完成。
 
 
 <!--
 ### Pod Name Label
 
-When the StatefulSet controller creates a Pod, it adds a label, `statefulset.kubernetes.io/pod-name`, 
-that is set to the name of the Pod. This label allows you to attach a Service to a specific Pod in 
+When the StatefulSet {{< glossary_tooltip term_id="controller" >}} creates a Pod,
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/statefulset.md]: 此行英文发生变化，请更新译文

File: content/zh/docs/concepts/workloads/controllers/statefulset.md [hunk 2]
```diff
@@ -397,9 +406,40 @@ update, roll out a canary, or perform a phased roll out.
...
+
+After reverting the template, you must also delete any Pods that StatefulSet had
+already attempted to run with the bad configuration.
+StatefulSet will then begin to recreate the Pods using the reverted template.
+-->
+#### 强制回滚 {#forced-rollback}
+
+在默认 [Pod 管理策略](#pod-management-policies)(`OrderedReady`) 时使用 [滚动更新](#rolling-updates) ，可能进入需要人工干预才能修复的损坏状态。
+
+如果将 `Pod template` 更新为永不运行或就绪的配置（例如，由于错误的二进制文件或应用程序级配置错误），StatefulSet 将停止回滚并等待。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/statefulset.md]:
```suggestion
如果更新后 Pod 模板配置进入无法运行或就绪的状态（例如，由于错误的二进制文件或应用程序级配置错误），StatefulSet 将停止回滚并等待。
```

File: content/zh/docs/concepts/workloads/controllers/statefulset.md [hunk 3]
```diff
@@ -397,9 +406,40 @@ update, roll out a canary, or perform a phased roll out.
...
+
+在这种状态下，仅将 `Pod template` 还原为正确的配置是不够的。由于 [已知问题](https://github.com/kubernetes/kubernetes/issues/67250)，
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/statefulset.md]:
```suggestion
在这种状态下，仅将 Pod 模板还原为正确的配置是不够的。由于 [已知问题](https://github.com/kubernetes/kubernetes/issues/67250)，
```

File: content/zh/docs/concepts/workloads/controllers/statefulset.md [hunk 4]
```diff
@@ -397,9 +406,40 @@ update, roll out a canary, or perform a phased roll out.
...
+StatefulSet 将继续等待损坏状态的 Pod 准备就绪（永远不会发生），然后再尝试将其恢复为正常工作配置。
+
+恢复模板后，还必须删除 StatefulSet 所具有，并且已经尝试使用错误的配置运行的 `Pod`。然后，StatefulSet 将开始使用被还原的模板来重新创建 Pod。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/statefulset.md]:
```suggestion
恢复模板后，还必须删除 StatefulSet 尝试使用错误的配置来运行的 Pod。这样，StatefulSet 才会开始使用被还原的模板来重新创建 Pod。
```

**Comment by xichengliudui** [comment]
/lgtm
/approve
@lpf7551321 thanks!

**Pull request merged**